tools/version_check.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+# For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+set -euo pipefail
+#set -o posix;
+LC_CTYPE="C"
```

Comment:
You don't need to set it per se but it solves a strange corner case.  Specifically, it's possible to define what constitutes `[[:digit:]]` and the like.  Why not just use `[0-9]`?  Funny thing... you can put other characters between 0 and 9 in your system character set or even put 9 before 0.

So while setting it for the script ensures proper function, I doubt anyone will ever run into this issue, making it more a matter of technical correctness because it could _hypothetically_ be important.